### PR TITLE
Release

### PR DESCRIPTION
**Pull request by OR13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,15 +112,15 @@ Once you have an instance of the Sidetree class with the suitable adapters, you
...
+const { Sidetree, MnemonicKeySystem } = require("@transmute/element-lib");
 
 // Instantiate the Sidetree class
 const element = new Sidetree(/* See previous section for how to initialize the Sidetree class*/);
 
 // Generate a simple did document model
 const mks = new MnemonicKeySystem(MnemonicKeySystem.generateMnemonic());
-const primaryKey = await mks.getKeyForPurpose('primary', 0);
-const recoveryKey = await mks.getKeyForPurpose('recovery', 0);
+const primaryKey = await mks.getKeyForPurpose("primary", 0);
```
Comment (gjgd) [inline README.md:122]: I think our convention is to use single quotes in this project

Comment (OR13) [reply README.md:122]: This is a readme file... and it was corrected based of the README formatting process.

Comment (OR13) [reply README.md:122]: makes me wonder why the format would be different, but not enough to want to fix it :)

Comment (gjgd) [reply README.md:122]: Ok!

**Pull request merged**